README.md

```diff
@@ -1 +1,160 @@
-# webpack-todo
\ No newline at end of file
+# Portfolio
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="Images\html_css.png" alt="logo" width="140"  height="auto" />
+  <br/>
```

Comment:
- [OPTIONAL] I'll suggest you remove this Microverse logo as it was placed here as a placeholder. The image is currently broken and it makes your readme look a bit unprofessional. :+1: :smile: